packages/@okta/vuepress-site/docs/guides/token-inline-hook/check-against-store/nodejs/check-patients.md

```diff
@@ -0,0 +1,35 @@
+```JavaScript
+// Example Patients Data Store
+
+const patients = [
+  {
+    username: 'michelle.test@example.com',
+    ExternalServicePatientID: '1235',
+  },
+  {
+    username: 'raj.gupta@example.com',
```

Comment:
In the Glitch example, the email for Patient ID 6789 is actually "bob.uncle@example.com" instead of "raj.gupta@example.com" - let's make the docs and Glitch code snippets consistent if possible.

Reply:
thanks, agree. Updated glitch example to match doc.